src/ai/backend/manager/services/model_serving/services/model_serving.py

```diff
@@ -205,6 +205,11 @@ async def create(self, action: CreateModelServiceAction) -> CreateModelServiceAc
                     if variant_def.environment:
                         action.creator.image = variant_def.environment.image
                         action.creator.architecture = variant_def.environment.architecture
+                    if variant_def.environ:
+                        if action.creator.config.environ:
+                            action.creator.config.environ.update(variant_def.environ)
+                        else:
+                            action.creator.config.environ = variant_def.environ
```

Comment:
What is the difference between environment and environ?

Reply:
`environ` is not related to `environment`